frontend/src/metabase/admin/permissions/components/PermissionsGrid.jsx

```diff
@@ -259,7 +260,7 @@ const EntityRowHeader = ({ entity, type }) =>
             <Icon name={type} className="absolute" style={{ left: -28 }} />
             <h4>{entity.name}</h4>
         </div>
-        { entity.subtitle &&
+        { type !== 'database' && entity.subtitle &&
```

Comment:
Instead of this you should just remove the subtitle from here https://github.com/metabase/metabase/blob/014a58bc65002d7438de562355019e326c7f48cb/frontend/src/metabase/admin/permissions/selectors.js#L245